api/spec/v1.yaml

```
@@ -1614,6 +1681,71 @@ components:
       description: |
         A ParaTime block.
 
+    RuntimeEventList:
+      type: object
+      required: [events]
+      properties:
+        events:
+          type: array
+          items:
+            $ref: '#/components/schemas/RuntimeEvent'
+      description: |
+        A list of runtime events.
+
+    RuntimeEvent:
+      type: object
+      required: [round, type, body]
+      properties:
+        round:
+          type: integer
+          format: int64
+          description: The block height at which this event was generated.
+          example: *runtime_block_round_1
+        tx_hash:
+          type: string
+          nullable: true
+          description: |
+            Hash of this event's originating transaction
+            Absent if the event did not originate from a transaction.
+          example: *tx_hash_1
+        type: 
+          description: The type of the event.
+          $ref: '#/components/schemas/RuntimeEventType'
+        body:
+          type: object
+          description: |
+            The decoded event contents. This spec does not encode the many possible types;
+            instead, see [the Go API](https://pkg.go.dev/github.com/oasisprotocol/oasis-sdk/client-sdk/go/modules).
+            This object will conform to one of the `*Event` types two levels down
+            the hierarchy, e.g. `MintEvent` from `Event > accounts.Event > MintEvent`,
+            OR `evm.Event` directly.
```

```suggestion
            the hierarchy (e.g. `MintEvent` from `accounts > Event > MintEvent`),
            OR `evm > Event`.
```

this better describes what they see / what they have to click if they follow the provided Go API link above.